### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by OscarFMdev**
# PR description
In this branch I:

- Added webpack to the project.
- Created the list structure.
- Created a function to check for the array of tasks and display them.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    index: 2,
+    description: 'World',
+    completed: true,
+  },
+  {
+    index: 3,
+    description: '!!!!',
+    completed: true,
+  },
+];
```
Comment (brytebee) [inline src/index.js]: - [ ] **Required:** You have rendered the list excellently to the DOM, however, you are required to render them in the order of their index.

**_Hint:_** Consider `Array.sort((a, b) => a.index - b.index`

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @OscarFMdev👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (OscarFMdev) [reply src/index.js]: @brytebee yes, I forgot that part, thank you for reviewing my code!

**Review by codecaiine — APPROVED** [review]
Hi @OscarFMdev , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Great project implementation :heavy_check_mark: 
- No linters errors :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**